### ISSUE TITLE
install script restore appears not to work

### ISSUE THREAD

**Issue by antoine-gallix** [DreymaR/BigBagKbdTrixXKB#21]
using the restore option of the install script, files installed by it previously are still there after the operation. I also don't seem to be able to remove them neither by reinstalling the package that own the keyboard config in my system. I mean, I like those changes up to now, but I was trying a complete reinstall of xkb and the trick bag as part of a debug investigation.

**Comment by antoine-gallix** [comment]
I think I haven't been precise enough in my original post. In a more systematic form, what I see as an issue is the following.

state 0: I have my original xkb data files

state 1: after installing the bag of trick with the script, I can see a backup, and new files in the xkb-data directory. new options are available in the menu.

I was expecting that running the script with the restore option would put me back in state 0. Apparently it does not, because I can still see the newly installed files after the restore, in the xkb data dir.

I was also expecting that reinstalling the xkb system package would also perform the same task of reseting the xkb data dir, but eventhough the installation is successful, the files installed by the bag of trick script are still there.

I'm not well versed enough in bash incantations to fully understand the script code but it definitely seems more complicated than replacing the xkb dir by it's last backup dir. I seems to do something file by file.

I solve the problem by deleting the xkb data dir and copying the backup in its place. But the operation is a bit frightening and I would have expected the install script to deinstall itself cleanly. In amy case, there is a problem or with my expectation of the effect of restore, or with the restore script.

**Comment by DreymaR** [comment]
It seems we have different goals then.

What I do is overwrite some files in the xkb dir. I keep a backup of the files I overwrite, and with the restore option I can reinstate the original files.

Neither I nor the xkb install is concerned with whatever else may be in that dir, as people may have their own files (like my symbols/extend file). I would not want to overwrite anything except the files I have changed, when restoring.

So basically, if you want to delete the backup dir then do so manually. The xkb install will only write what it needs, and so will my install. None of us want to replace the whole directory because that might ruin someone else's additions.
